### ISSUE TITLE
Implement mandate administration UI

### ISSUE THREAD

**Issue by noahmay** [Dsek-LTH/member-page#78]
## Short description of the feature
- Create a UI for creating a mandate
- Create a UI for updating/deleting a mandate

Do this issue depend on another issue? Write Depends on #38

[x] Relates to frontend
[ ] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service
